enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/MobileTokenService.java

```diff
@@ -262,7 +262,7 @@ public Response operationReject(
             return new Response();
         } else {
             final OperationDetailResponse operation = rejectResponse.getOperation();
-            handleStatus(operation.getStatus());
+            handleStatus(operation);
```

Comment:
and should not we be consistent with raising OPERATION_FAILED as default when an operation failed?